src/blockdev.rs

```diff
@@ -809,6 +809,56 @@ pub fn lsblk_single(dev: &Path) -> Result<HashMap<String, String>> {
     Ok(devinfos.remove(0))
 }
 
+fn lsblk_all() -> Result<Vec<HashMap<String, String>>> {
+    let mut cmd = Command::new("lsblk");
+    cmd.arg("--noheadings")
```

Comment:
Optional: I find these slightly more readable as `cmd.args(&["--noheadings", "--nodeps", "--list"])` etc.

(There's also my https://docs.rs/sh-inline/0.1.0/sh_inline/macro.bash_command.html macro which can make this just `bash_command!("lsblk --noheadings --nodeps --list --paths --output NAME")` safely)